operator/src/main/resources/scripts/utils.sh

```diff
@@ -395,3 +395,26 @@ checkWebLogicVersion()
   fi
   return 0
 }
+
+
+#
+# getAdminUrl
+#   purpose: Get the ADMIN_URL used to connect to the admin server internally, e.g. when starting a managed server
+#   sample:
+#     ADMIN_URL=$(getAdminUrl)
+#
+function getAdminUrl() {
+  admin_protocol="http"
```

Comment:
add keyword 'local' before admin_protocol

Reply:
(only needed for the first  use of the local variable, other functions should do this with local variables too but I wouldn't be surprised if we haven't caught them all)

Reply:
Sure.